Services/TokenAcquisition.cs

```diff
@@ -63,9 +63,10 @@ public class TokenAcquisition : ITokenAcquisition
         /// This constructor is called by ASP.NET Core dependency injection
         /// </summary>
         /// <param name="options">Options to configure the application</param>
-        public TokenAcquisition(ITokenCacheProvider tokenCacheProvider, IOptions<AzureAdOptions> options)
+        public TokenAcquisition(ITokenCacheProvider tokenCacheProvider, IConfiguration configuration)
```

Comment:
Interestingly (and I don't know why), using IOptions<AzureADOptions> did not work
I had to explicitly bind the section?